### ISSUE TITLE
ibazel does not pass arguments to Bazel

### ISSUE THREAD

**Issue by endobson** [bazelbuild/bazel-watcher#13]
I want to run my tests in parallel and ibazel adds `--test_output=streamed` to the args when running `bazel test`, which is unexpected and undocumented. I would expect it to just run exactly the same had the command line been `bazel test`.

**Comment by achew22** [comment]
@endobson Sorry for the really slow response. The presumption here is that you would like all blaze test args to be sent along to later commands but that we would need to filter the args list to only take things that are shaped like a target for query. That seems possible. Especially given the refactor I did recently. Is this a thing you would be interested in taking on?

**Comment by kamalmarhubi** [comment]
Relatedly: ibazel cannot pass _any_ arguments to bazel. I spent some time trying to understand why `ibazel run //path/to:some_binary -- --an-arg` wasn't working.

I'm including this here because it's related. The issue could maybe be renamed to support passing arguments to bazel.

**Comment by yzhao1012** [comment]
This does not seem work as expected. 

ibazel test //path/to:target -- --test_output=all

This would not make the test to produce all output as the following does:

bazel test //path/to:target --test_output all

**Comment by achew22** [comment]
When you put the extra `--` in there, all flags past that are directly passed to the binary. The more equivilent command would be `bazel test //path/to:target -- --test_output=all`. Can you try running

```
ibazel test --test_output=all //path/to:target
```

and tell me if that works?

Also, in the future it would be nice if you could open a new issue instead of commenting on a closed one.
